go-controller/pkg/node/controllers/egressip/egressip.go

```diff
@@ -281,18 +292,12 @@ func (c *Controller) Run(stopCh <-chan struct{}, wg *sync.WaitGroup, threads int
 			if err = c.iptablesManager.EnsureRule(utiliptables.TableMangle, utiliptables.ChainPrerouting, utiliptables.ProtocolIPv6, iptSaveMarkRule); err != nil {
 				return fmt.Errorf("failed to create rule in chain %s to save pkt marking: %v", utiliptables.ChainPrerouting, err)
 			}
-		}
-	}
-	if ovnconfig.Gateway.Mode == ovnconfig.GatewayModeLocal {
-		// If dst is a node IP, use main routing table and skip EIP routing tables
-		if err = c.ruleManager.Add(getNodeIPFwMarkIPRule()); err != nil {
-			return fmt.Errorf("failed to create IP rule for node IPs: %v", err)
-		}
-		// The fwmark of the packet is included in reverse path route lookup. This permits rp_filter to function when the fwmark is
-		// used for routing traffic in both directions.
-		stdout, _, err := util.RunSysctl("-w", "net.ipv4.conf.all.src_valid_mark=1")
```

Comment:
hmm we don't want to do this for v6?

Reply:
`src_valid_mark` doesn't exist for ipv6

Reply:
ack so same as https://github.com/ovn-org/ovn-kubernetes/pull/3006#discussion_r917062397